File: packages/mlmpfr.4.0.2+dune/utils/mpfr_version.ml
(* ocamlc -I ../_build/src -custom ../_build/src/mpfr.cmo ../utils/mpfr_version.ml ../_build/src/mlmpfr_stubs.o -ccopt -L/usr/local/lib -cclib -lmpfr *)

module M = Mpfr

let _ =
  Printf.printf "%s\n" (M.get_version ())

File: packages/mlmpfr.4.1.0/utils/mpfr_version.c
// gcc -L /usr/local/lib -o mpfr_version mpfr_version.c -lmpfr
// LD_LIBRARY_PATH=/usr/local/lib mpfr_version

#include <stdio.h>
#include <mpfr.h>

int main (void)
{
  printf ("%s\n", mpfr_get_version ());

  return 0;
}
